### PR TITLE
Add readability: fixes 29

### PR DESCRIPTION
**Pull request by pippinlee**
- left align all cells
- no background colour
- add green for valid options cells
- top align "Validation" cells

### CODE REVIEW

File: examples/index.less [hunk 1]
```diff
@@ -153,35 +153,52 @@ header.links {
...
+    }
+    th {
+      border-bottom: none;
       text-align: center;
     }
+    tr {
+      height: 80px;
+    }
+    td:nth-child(1) {
+      vertical-align: top;
```
Comment (k88hudson) [inline examples/index.less]: you probably want to put this for all `td`

File: examples/index.less [hunk 2]
```diff
@@ -153,35 +153,52 @@ header.links {
...
+      padding-top: 19px;
```
Comment (k88hudson) [inline examples/index.less]: why different padding here than the other tds?

File: examples/index.less [hunk 3]
```diff
@@ -153,35 +153,52 @@ header.links {
...
+      code {
+        display: block;
+        vertical-align: top;
+        color: #000;
```
Comment (k88hudson) [inline examples/index.less]: `code { color: #000; }` is probably the only thing you need to declare under td:nth-child(1)

File: examples/index.less [hunk 4]
```diff
@@ -153,35 +153,52 @@ header.links {
...
+      }
+    }
+    td:nth-child(2), td:nth-child(3) {
+      vertical-align: top;
+      code {
+        display: block;
+        margin-bottom: 10px;
+        margin-top: 10px;
```
Comment (k88hudson) [inline examples/index.less]: why does this need margin-top?

File: examples/index.less [hunk 5]
```diff
@@ -153,35 +153,52 @@ header.links {
...
-      margin-top: 10px;
-      padding: 5px;
+      background-color: transparent;
+    }
+    th {
+      border-bottom: none;
       text-align: center;
     }
+    tr {
+      height: 80px;
```
Comment (k88hudson) [inline examples/index.less]: why are you declaring a height here?

**Comment by k88hudson** [comment]
There are few things you could probably clean up in terms of merging styles together. Also, I think the labels in the heading row are too tall/need to be left aligned

**Comment by pippinlee** [comment]
Explanation:

```
tr:nth-child(1) {
  td:nth-child(4) {
    border-top: none;
  }
}
.
.
.
td:nth-child(4) {
  display: block;
  width: 200px;
  border: none
  border-top: 1px solid #DDD;
}
```

This style targets the last column of the second row. Why do we need it?

This is mainly due to our need to make the last column 200px wide. To do this we need to add `display: block` to the 4th column.

The way our markdown file is converted to html means that each 4th column cell is a `<td>` element. The other thing to note is our text isn't within a child element like the other 3 columns are within a `<code>` element.

When we use `display: block` this uses the previously defined border for `<td>` and creates on extra border around all 4th column elements, so when we add a `borders: none` value we must also add a `border-top: 1px solid #DDD` value to give the 4th column `<td>` elements a top border.



The side-effect of this means our 2nd row 4th column cell has an extra 1px border on top. So we must use the above quoted styling to negate this side effect.

This gives us the desired result of good readability in the last column with a width of 200px, while also giving us the proper borders in each of the 4th column cells, and in the first row heading cells.

**Comment by k88hudson** [comment]
Nice! Last remaining issue is that the `th` text is still centered. Looking awesome :+1:

**Comment by pippinlee** [comment]
Oops, sorry. Updated.

**Comment by k88hudson** [comment]
Thanks!

**Pull request merged**